### DISCUSSION TITLE
Add a local ansible file

### DISCUSSION THREAD

**Discussion by fperillo** [vmware/packer-examples-for-vsphere#792] [Ideas]
I made the error to locally modify the ansible files provided by this project and now I'm really afraid to pull the new version, since the merge can be daunting.

So I'm wondering if it would be possible to add something that allows to include local files, untracked or guaranteed to be empty, from this project.

For each of the roles, add a empty local.yml that is always included in the playbook but is modified only locally, so that new merge won't break it.

Or a method to invoke a local playbook, before finalizing the template. Unfortunately I can't officially use ansible after creating the template, at least not yet, so my request.

Thank you

**Comment by tenthirtyam** [comment]
I'll let @gcblack mull this one over and comment.

**Comment by fperillo** [comment]
I simply added a new role called "local" and created the corresponding directory tree. This is a quick and dirty solution :-)

I then spent a full day adapting my old configuration to the new structure :-(

**Reply by tenthirtyam** [reply]
🙃
